### PR TITLE
minor doc fixes

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: documentation/asciidoc/topics/proc_building_and_running.adoc [hunk 1]
```diff
@@ -24,5 +24,5 @@ $ git clone {repository_ssh}
 +
 [source,bash,options="nowrap",subs=attributes+]
 ----
-$ mvn clean install -DskipTests=true
+$ mvn -s /path/to/maven-settings.xml clean install -DskipTests=true
```
Comment (pruivo) [inline documentation/asciidoc/topics/proc_building_and_running.adoc:27]: the maven-settings.xml is only required for downstream.

Comment (oraNod) [reply documentation/asciidoc/topics/proc_building_and_running.adoc]: thanks @pruivo I've fixed this so the maven-settings.xml arg is downstream only.

File: infinispan-remote/cache-admin-api/README.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ on Infinispan servers.
 . Build and run the Hot Rod Java client as follows:
 
 ----
-$ mvn clean package
```
Comment (pruivo) [inline infinispan-remote/cache-admin-api/README.adoc:12]: this is a README file and not part of the docs. is the attribute going to be replaced?

**Comment by pruivo** [comment]
integrated! thanks @oraNod

**Pull request merged**